### PR TITLE
Add some missing functions to fs-extra

### PR DESCRIPTION
**Pull request by villesau**
fs-extra has all the fs functions. I added some of which our CLI relies on. copied from here: https://github.com/facebook/flow/blob/master/lib/node.js

see: https://github.com/jprichardson/node-fs-extra

### CODE REVIEW

**Comment by pascalduez** [comment]
Thanks, I have a branch to improve `fs-extra` and add node functions, but sort of procrastinated on it :)

File: definitions/npm/fs-extra_v7.x.x/flow_all/fs-extra_v7.x.x.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 declare module "fs-extra" {
-  import type { Stats } from "fs";
+  import type { Stats, ReadStream, WriteStream } from "fs";
```
[the remark this answers is not in the document]
Comment (pascalduez) [reply definitions/npm/fs-extra_v7.x.x/flow_all/fs-extra_v7.x.x.js]: Not the ones from Flow core, like Node.js, React, DOM etc.

[the remark this answers is not in the document]
Comment (villesau) [reply definitions/npm/fs-extra_v7.x.x/flow_all/fs-extra_v7.x.x.js]: If you import them _outside_ module declaration, that's `any`.

**Comment by langri-sha** [comment]
Quick question: `fs-extra` has a default export. It seems it would be correct to remove all the export directives, as it's done for some built-in, like `fs` itself :thinking:?

**Comment by villesau** [comment]
@langri-sha if you don't define any exports, all the declarations are exported. I think this is due to legacy reasons when Flow didn't have `export` keyword in libdefs.

**Pull request merged**